React/todoApp/src/components/Form.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./Form.module.css";
+
+export default function Form({ setTodos }) {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const value = e.target.todo.value.trim();
+    if (!value) return;
+
+    setTodos(prev => [
+      ...prev,
+      { id: crypto.randomUUID(), title: value, is_completed: false }
```

Comment:
In JS, we prefer CamelCases
```suggestion
      { id: crypto.randomUUID(), title: value, isCompleted: false }
```